### PR TITLE
Replace `javax.security.auth.Policy` with `java.security.Policy`

### PR DESCRIPTION
**Pull request by BhavanaPidapa**
## What's changed?
image

## What's your motivation?
I used org.openrewrite.java.ChangeType for the recipe RemovedPolicy

## Anyone you would like to review specifically?
@cjobinabo @timtebeek 

## Have you considered any alternatives or workarounds?
Since it was not possible to test the recipe. I created sample files, created a local plugin using
**./gradlew publishToMavenLocal** then ran it in the Java8 Sample App using **mvn rewrite:dryRun**

Attaching the rewrite.patch file
[rewrite.patch](https://github.com/user-attachments/files/15811634/rewrite.patch)

### Checklist
- [X] I've added unit tests to cover both positive and negative cases
- [X] I've read and applied the [recipe conventions and best practices](https://docs.openrewrite.org/authoring-recipes/recipe-conventions-and-best-practices)
- [X] I've used the IntelliJ IDEA auto-formatter on affected files

### CODE REVIEW

File: src/main/resources/META-INF/rewrite/java-version-11.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2022 the original author or authors.
+# Copyright 2024 the original author or authors.
```
Comment (timtebeek) [inline src/main/resources/META-INF/rewrite/java-version-11.yml:2]: For any future PR: no need to update these copyright headers; we typically keep the year on when they were first introduced.

**Review by timtebeek — APPROVED** [review]
Neat addition @bhavanaPidapa, thanks for adding the recipe and verifying it locally. Indeed it can be difficult to unit test classes removed in newer Java versions, but this should be ok.

One small suggestion to update the phrasing of the displayed name and description, but otherwise good to merge from my end. I'll leave it up to @cjobinabo to do a final check and merge.

**Review by cjobinabo — APPROVED** [review]

**Comment by cjobinabo** [comment]
I've gone ahead and commited the suggestion. Thats for looking at this @timtebeek

**Comment by timtebeek** [comment]
Great! Then feel free to merge whenever suits you best. :)

**Pull request merged**